infrastructure/module/eks3/ebs_csi.iam.tf

```diff
@@ -0,0 +1,14 @@
+module "ebs_csi_iam_role" {
+  source = "terraform-aws-modules/iam/aws//modules/iam-role-for-service-accounts-eks"
+
+  role_name = "AmazonEKS_EBS_CSI_DriverRole_igork"
+
+  role_policy_arns = {AmazonEBSCSIDriverPolicy:"arn:aws:iam::aws:policy/service-role/AmazonEBSCSIDriverPolicy"}
+
+  oidc_providers = {
+    main = {
+      provider_arn               = module.eks.oidc_provider_arn
+      namespace_service_accounts = ["system:serviceaccount:kube-system:ebs-csi-controller"]
```

Comment:
This is not good, it should be:

**kube-system:ebs-csi-controller-sa**

Check the docs:

https://docs.aws.amazon.com/eks/latest/userguide/csi-iam-role.html